### PR TITLE
Recursively delete scripts dir when removing backup. 

### PR DESCRIPTION
**Pull request by armiller**
I realize there is no tests at the moment but this cookbook needs a lot of work. This is a small change so it should be fine. @osuosl-cookbooks/chefs

### CODE REVIEW

**Comment by ramereth** [comment]
@armiller is there anyway to add an integration test for this?

**Comment by armiller** [comment]
there are currently no tests for this cookbook so I'm not sure how long it would take...

**Comment by ramereth** [comment]
@armiller take a look at [osl-backup](https://github.com/osuosl-cookbooks/osl-backup), I believe it already has some examples there that we could import and adapt. But let me know if its more involved than that.

**Comment by fahlmant** [comment]
@codymalick Still working on this?

**Comment by ramereth** [comment]
@osuosl-cookbooks/chefs I think this is finally ready for a final review

**Comment by armiller** [comment]
🎉  Still a contributor after a year! Oh yeah 😎

**Comment by armiller** [comment]
and my last open pr with the OSL 😢

File: spec/rdiff-backup-test/create_server_spec.rb [hunk 1]
```diff
@@ -126,6 +148,18 @@
         )
       end
       it do
+        expect(chef_run).to create_file(
+          '/home/rdiff-backup-server/exclude/192.168.60.12/_test2'
+        ).with(
+          owner: 'rdiff-backup-server',
+          group: 'rdiff-backup-server' || 'rdiff-backup-server',
```
Comment (knightsamar) [inline spec/rdiff-backup-test/create_server_spec.rb]: What does this line mean?

Comment (ramereth) [inline spec/rdiff-backup-test/create_server_spec.rb]: I'm not sure, I just copied and pasted it from another part. I can likely remove it.

Comment (knightsamar) [inline spec/rdiff-backup-test/create_server_spec.rb]: `||` makes me think it was meant to be something different. It  it's no longer useful, so please remove one.

**Comment by ramereth** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.0.10.  Have a nice day!

**Pull request merged**